things/api.py

```diff
@@ -86,16 +85,26 @@ def tasks(uuid=None, include_items=False, **kwargs):  # noqa: C901
         - `tag == True`, only include tasks _with_ tags.
         - `tag == None` (default), then include all tasks.
 
-    start_date : bool or None, optional
+    start_date : bool, str or None, optional
         - `start_date == False`, only include tasks _without_ a start date.
         - `start_date == True`, only include tasks _with_ a start date.
+        - `start_date == 'future'`, only include tasks with a future start date.
+        - `start_date == 'past'`, only include tasks with a past start date.
         - `start_date == None` (default), then include all tasks.
 
-    deadline : bool or None, optional
+    deadline : bool, str or None, optional
         - `deadline == False`, only include tasks _without_ a deadline.
         - `deadline == True`, only include tasks _with_ a deadline.
+        - `deadline == 'future'`, only include tasks with a deadline in the future.
+        - `deadline == 'past'`, only include tasks with a deadline in the past.
         - `deadline == None` (default), then include all tasks.
 
+    suppressed : bool or None, optional
```

Comment:
This terminology doesn't seem to be used anywhere in the app, only in the database.
To clarify: when is `dueDateSupressionDate` set? Is it only for tasks moved to the Inbox? Do we absolutely need to expose it, or are there other ways to deduce these tasks?

Reply:
From what I saw this seems to be an internal hack only for overdue tasks moved out of the inbox. I only need it for `unconfirmed_overdue_tasks`. So we can remove the documentation from the `tasks` method.

Reply:
I played around a bit with the Things app. I now better understand this one. You can move overdue to-dos (and maybe also projects?) from the Today view to Inbox, Anytime, or Someday and a dueDateSuppressionDate will be set.

Proposal:
Leave it, but rename it to `deadline_suppressed` for consistency and readability. Also explain, this is for overdue tasks that were moved from Today to Inbox, Anytime, or Someday.